src/iidm/util/SV.cpp

```diff
@@ -0,0 +1,191 @@
+    return v2 * 3.0 * std::conj(i2);
+}
+
+std::complex<double> SV::computeS2(const std::complex<double>& y, const std::complex<double>& z, double rho) const {
+    return computeS2(y, std::complex<double>(), z, rho);
+}
+
+std::complex<double> SV::computeU2(const std::complex<double>& y1, const std::complex<double>& /*y2*/, const std::complex<double>& z, double rho) const {
+    std::complex<double> v1p = m_v1 * rho;
+    std::complex<double> i1p = m_i1 / rho;
+
+    std::complex<double> i2p = i1p - (y1 * v1p);
+    std::complex<double> v2 = v1p - (z * i2p);
+    return v2 * std::sqrt(3.0);
+}
+
+std::complex<double> SV::computeU2(const std::complex<double>& y, const std::complex<double>& z, double rho) const {
+    return computeU2(y, std::complex<double>(0, 0), z, rho);
+}
+
+double SV::getA() const {
+    return m_a;
+}
+
+double SV::getB(const TwoWindingsTransformer& twt) {
+    return twt.getB() * (twt.hasRatioTapChanger() ? 1 + twt.getRatioTapChanger().getCurrentStep().getB() / 100 : 1.0);
+}
+
+double SV::getG(const TwoWindingsTransformer& twt) {
+    return twt.getG() * (twt.hasRatioTapChanger() ? 1 + twt.getRatioTapChanger().getCurrentStep().getG() / 100 : 1.0);
+}
+
+double SV::getP() const {
+    return m_p;
+}
+
+double SV::getQ() const {
+    return m_q;
+}
+
+double SV::getR(const TwoWindingsTransformer& twt) {
+    return twt.getR() * (twt.hasRatioTapChanger() ? 1 + twt.getRatioTapChanger().getCurrentStep().getR() / 100 : 1.0);
+}
+
+double SV::getRho(const TwoWindingsTransformer& twt) {
+    return twt.getRatedU2() / twt.getRatedU1() * (twt.hasRatioTapChanger() ? 1 + twt.getRatioTapChanger().getCurrentStep().getRho() / 100 : 1.0);
+}
+
+double SV::getU() const {
+    return m_u;
+}
+
+double SV::getX(const TwoWindingsTransformer& twt) {
+    return twt.getX() * (twt.hasRatioTapChanger() ? 1 + twt.getRatioTapChanger().getCurrentStep().getX() / 100 : 1.0);
+}
+
+SV SV::otherSide(double r, double x, double g, double b, double rho) const {
+    std::complex<double> z(r, x);
+    std::complex<double> y(g, b);
+    std::complex<double> u2 = computeU2(y, z, rho);
+    std::complex<double> s2 = computeS2(y, z, rho);
+    return {-s2.real(), -s2.imag(), std::abs(u2), std::arg(u2) * stdcxx::toDegrees};
+}
+
+SV SV::otherSide(double r, double x, double g1, double b1, double g2, double b2, double rho) const {
+    std::complex<double> z (r, x);
+    std::complex<double> y1(g1, b1);
+    std::complex<double> y2(g2, b2);
+    std::complex<double> u2 = computeU2(y1, y2, z, rho);
+    std::complex<double> s2 = computeS2(y1, y2, z, rho);
+    return {-s2.real(), -s2.imag(), std::abs(u2), std::arg(u2) * stdcxx::toDegrees};
+}
+
+SV SV::otherSide(const TwoWindingsTransformer& twt) const {
+    return otherSide(getR(twt), getX(twt), getG(twt), getB(twt), getRho(twt));
+}
+
+SV SV::otherSide(const Line& line) const {
+    return otherSide(line.getR(), line.getX(), line.getG1() + line.getG2(), line.getB1() + line.getB2(), 1);
+}
```

Comment:
I have a doubt about the implementation of this method: I asked to @zamarrenolm and waiting for his feedback. From my opinion, we should not add g1 and g2 (same for b1 and b2) but call `otherSide(r, x, g1, b1, g2, b2, 1)`